Makefile

```diff
@@ -386,7 +410,7 @@ ifeq ($(TARGETSYSTEM),WINDOWS)
   BINDIST = $(W32BINDIST)
   BINDIST_CMD = $(W32BINDIST_CMD)
   ODIR = $(W32ODIR)
-  ifdef DYNAMIC_LINKING
+  ifeq ($(DYNAMIC_LINKING), 1)
```

Comment:
Make consistent with description at the beginning of the file (sometimes is required on MSYS2).